### PR TITLE
Prep for 0.9.21

### PR DESCRIPTION
**Pull request by odow**
@blegat can you run the SolverTests?

### CODE REVIEW

**Review by blegat — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+For a detailed list of the closed issues and pull requests from this release,
+see the [tag notes](https://github.com/jump-dev/MathOptInterface.jl/releases/tag/v0.9.21).
+
+- Added `supports_shift_constant`.
+- Improve performance of bridging quadratic constraints.
+- Add precompilation statements.
+- Large improvements to the documentation.
+- Fix a variety of inference issues, benefiting precompilation and reducing
+  initial latency.
+- `RawParameter`s are now ignored when resettting a `CachingOptimizer`.
```
Comment (GregPlowman) [inline NEWS.md:16]: typo

**Comment by odow** [comment]
SolverTest failures:

- [ ] SumOfSquares
- [ ] PolyJuMP
- [ ] COSMO
- [ ] OSQP: bug in their tests https://github.com/oxfordcontrol/OSQP.jl/pull/95
- [ ] ProxSDP
- [ ] SDPA
- [ ] SDPAFamily
- [x] Tulip: uses Convex.jl in tests which doesn't support new MOI.
- [ ] Xpress

**Comment by odow** [comment]
I don't think there's any blocking issue here. No solver is broken, just their tests (and some just because of the Julia 1.6 release). PRs are open to fix.

**Comment by blegat** [comment]
Not that if we're just fixing tests, a new release not strictly necessary as SolverTests test against master

**Comment by odow** [comment]
PolyJuMP still failing:
image
https://github.com/blegat/SolverTests/runs/2410104837

But I don't think that's a blocker. Looks like just a dictionary change in the printing order or something.

**Comment by joaquimg** [comment]
Looking into ProxSDP

**Comment by odow** [comment]
@joaquimg did you make any progress? Alternatively, if you don't think it's a MOI breakage, let me know.

**Comment by joaquimg** [comment]
Fixed, update you list.

**Pull request merged**